src/tracks/ui/TextEditDelegate.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
Add header comments as in other headers describing the purpose of the file, and use
@file and @brief comments for Doxygen.